android/sdk/src/main/java/com/taobao/weex/ui/component/WXScroller.java

```diff
@@ -482,44 +485,45 @@ public void onScrollChanged(WXHorizontalScrollView scrollView, int x, int y, int
       scrollView.addView(mRealView, layoutParams);
       scrollView.setHorizontalScrollBarEnabled(false);
       mScrollerView = scrollView;
-        final WXScroller component = this;
-        final View.OnLayoutChangeListener listener = new View.OnLayoutChangeListener() {
-          @Override
-          public void onLayoutChange(View view, int left, int top, int right, int bottom, int oldLeft, int oldTop, int oldRight, int oldBottom) {
-            final View frameLayout = view;
-            scrollView.post(new Runnable() {
-              @Override
-              public void run() {
-                if (isLayoutRTL()) {
-                  int mw = frameLayout.getMeasuredWidth();
-                  scrollView.scrollTo(mw, component.getScrollY());
-                } else {
-                  boolean scrollToBegin = true;
-                  Object scrollToBeginOnLTR = getAttrs().get("scrollToBegin");
```

Comment:
`ScrollBegin` is added in https://github.com/apache/incubator-weex/pull/2165

Is there an reason to remove?

Reply:
Because it do not scroll to begin anymore.